src/api/landing-page/content-types/landing-page/schema.json

```diff
@@ -0,0 +1,73 @@
+{
+  "kind": "singleType",
+  "collectionName": "landing_pages",
+  "info": {
+    "singularName": "landing-page",
+    "pluralName": "landing-pages",
+    "displayName": "Landing page"
+  },
+  "options": {
+    "draftAndPublish": false
+  },
+  "pluginOptions": {
+    "i18n": {
+      "localized": true
+    }
+  },
+  "attributes": {
+    "headerText": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "string",
+      "required": false
+    },
+    "headerPhoto": {
+      "allowedTypes": [
+        "images"
+      ],
+      "type": "media",
+      "multiple": false,
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "required": false
+    },
+    "buttonText": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "string"
+    },
+    "buttonColor": {
+      "pluginOptions": {
```

Comment:
Should buttonColor be localised, and should it even be an option in the landing page Strapi type? Isn't it more of a styling thing that should be coded on the frontend side? (Or Strapi could have a separate type for colors/styling stuff across the entire website).

Reply:
This was mainly done to support different `headerImage`s for languages.